src/components/Header.js

```diff
@@ -58,12 +64,24 @@ class Header extends React.Component {
         {user && (
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav navbar className="ml-auto">
-              <Link route="userSettings" passHref>
-                <NavLink className="navbar-text">
-                  <FontAwesomeIcon icon={faUser} className="mr-2" />
-                  {userName}
-                </NavLink>
-              </Link>
+              <div className="d-flex justify-content-between">
```

Comment:
It was something I put in when I tried to add more space between the username and the logout button, but that is no longer relevant because I used "ml-3" in the button instead... deleting the div!